lib/rules/jsx-indent-props.js

```diff
@@ -78,62 +81,41 @@ module.exports = {
      * @param {ASTNode} node Node violating the indent rule
      * @param {Number} needed Expected indentation character count
      * @param {Number} gotten Indentation character count in the actual node/code
-     * @param {Object=} loc Error line and column location
      */
-    function report(node, needed, gotten, loc) {
```

Comment:
is the `loc` no longer required because eslint gets it off the node?

Reply:
It was never provided when calling the function - probably just a copy&paste leftover from somewhere else.